addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  _updateModel(model, infinityModel) {
```

Comment:
Yeah agree!  I was under the impression that the "pushObjects" to model content no longer works in new versions of Ember Data.  What version of Ember Data are you using?